core/src/epicli/data/common/ansible/playbooks/roles/postgres_exporter/tasks/main.yml

```diff
@@ -0,0 +1,111 @@
+---
+- name: Install pip3 [Debian]
+  apt:
+    name:
+      - libpq5
+      - python3-psycopg2
+      - python3-jmespath
+    state: present
+    install_recommends: no
+  when: ansible_os_family == 'Debian'
+
+- name: Install pip3 [RedHat]
+  yum:
+    name:
+      - libpq5
+      - python3-psycopg2
+      - python3-jmespath
+    state: present
+    install_recommends: no
+  when: ansible_os_family == 'RedHat'
+
+- name: Create postgres_exporter system group
+  group:
+    name: postgres_exporter
+    system: true
+    state: present
+
+- name: Create postgres_exporter system user
+  user:
+    name: postgres_exporter
+    system: true
+    shell: "/usr/sbin/nologin"
+    group: postgres_exporter
+    createhome: false
+
+- name: Set Postgres Exporter file name to install
+  set_fact:
+    exporter_file_name: "{{ specification.file_name }}"
+
+- name: Download Postgres Exporter binaries
+  include_role:
+    name: download
+    tasks_from: download_file
+  vars:
+    file_name: "{{ exporter_file_name }}"
+
+- name: Create /opt/postgres_exporter directory
+  become: yes
+  file:
+    path: "{{ item }}"
+    recurse: yes
+    owner: root
+    group: postgres_exporter
+    mode: 0755
+    state: directory
+  with_items:
```

Comment:
Is it supposed to be done so for 1 collection element?

Reply:
It's not. It's leftover after initial config when i planned to create more complicated structure